test/DynamoCoreTests/Setup.cs

```diff
@@ -20,7 +20,8 @@ public void RunBeforeAllTests()
             var resolutionPaths = new[]
             {
                 // These tests need "CoreNodeModels.dll" under "nodes" folder.
-                Path.Combine(moduleRootFolder, "nodes")
+                Path.Combine(moduleRootFolder, "nodes"),
+                Path.Combine(new DirectoryInfo(assemblyPath).Parent.Parent.Parent.FullName, "test", "packages")
```

Comment:
hmrmm?

Reply:
This will allow assemblies to resolve from the new packages folder I created. The packages folder will contain the assemblies required by test projects.

Reply:
this is to avoid mixing them into the bin folder?

Reply:
yes